src/utils/compute.ts

```diff
@@ -39,3 +43,44 @@ export async function transformComputeFormToServiceComputePrivacy(
 
   return privacy
 }
+
+export async function transformDDOToAssetSelection(
+  ddoList: DDO[],
+  metadataCacheUri: string,
+  selectedAlgorithms?: PublisherTrustedAlgorithm[]
+): Promise<AssetSelectionAsset[]> {
+  const source = axios.CancelToken.source()
+  const didList: string[] = []
+  const priceList: any = {}
+  ddoList.forEach((ddo: DDO) => {
+    const did: string = web3.utils
```

Comment:
missed that in the edit compute PR: `ddo.id` is the DID. No need to get it from the datatoken address.

Reply:
hmm true, i'll remove the extra logic